script.js

```diff
@@ -1,4 +1,74 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+  var spsOutput = ['scissors', 'paper', 'stone']
+  var setSpsOutput =  Math.floor(Math.random()*spsOutput.length)
```

Comment:
Usually when it comes to random numbers, it is safer to put it outside the main function, because you might want to reuse the random number generator for other selections
so if it is not outside the main function, the number will always be the same through the flow of the program